frontend/src/app/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card flex justify-content-center">
+  <p-menu [model]="items">
+    <ng-template pTemplate="item" let-item>
+      <ng-container *ngIf="item.route; else elseBlock">
```

Comment:
i would use the new control-flow syntax as it improves readability tremendously. 
https://angular.dev/guide/templates/control-flow

Reply:
Maybe you want to do this as its a good opportunity to learn new angular stuff.